framework/src/play/libs/OpenID.java

```diff
@@ -326,6 +330,16 @@ public static String discoverServer(String openid) {
         return server;
     }
 
+    private static Document getXml(String response) {
+        try {
+            InputSource source = new InputSource(new StringReader(response));
+            DocumentBuilder builder = XML.newDocumentBuilder();
```

Comment:
According to this, specifying the encoding for the InputSource has no effect for character streams. http://docs.oracle.com/javase/6/docs/api/org/xml/sax/InputSource.html#setEncoding(java.lang.String)